### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by zeinjaber**

### CODE REVIEW

**Review by umahasit — APPROVED** [review]

**Review by pedrosousa — APPROVED** [review]

File: products/waf/src/content/exposed-credentials-check/index.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ order: 3
 
 # Automated exposed credentials check (Beta)
 
-<Aside type='warning'>
```
Comment (kzaw20081) [inline products/waf/src/content/exposed-credentials-check/index.md:8]: yarwana1122

**Review by kzaw20081 — APPROVED** [review]
(redacted)

File: products/waf/src/content/exposed-credentials-check/index.md [hunk 2]
```diff
@@ -5,16 +5,16 @@ order: 3
...
+<Aside type='note'>
 
-This feature is only available for selected customers on an Enterprise plan.
+This feature is available to all paid plans.
```
Comment (kzaw20081) [inline products/waf/src/content/exposed-credentials-check/index.md:10]: support@classy.org

**Review by kzaw20081 — APPROVED** [review]
(redacted)

**Pull request merged**